core/src/main/thrift/client.thrift

```diff
@@ -325,64 +325,76 @@ service ClientService {
     2:trace.TInfo tinfo
     3:security.TCredentials credentials
     1:ConfigurationType type
+  ) throws (
+    1:ThriftSecurityException sec
   )
 
   map<string, string> getSystemProperties(
     1:trace.TInfo tinfo
     2:security.TCredentials credentials
+  ) throws (
+    1:ThriftSecurityException sec
   )
 
   TVersionedProperties getVersionedSystemProperties(
     1:trace.TInfo tinfo
     2:security.TCredentials credentials
+  ) throws (
+    1:ThriftSecurityException sec
   )
 
   map<string, string> getTableConfiguration(
     1:trace.TInfo tinfo
     3:security.TCredentials credentials
     2:string tableName
   ) throws (
-    1:ThriftTableOperationException tope
+    1:ThriftSecurityException sec
+    2:ThriftTableOperationException tope
```

Comment:
This (and similar occurrences) changes the ordinal for the ThriftTableOperationException. This is okay if this is new for 2.1, but I haven't checked.

Reply:
Good catch, I didn't think about that. I usually think about that for input parameters. I'll evaluate and change if necessary.